index.rst

```diff
@@ -147,176 +246,121 @@ The white light source and monochromator sit on a table surrounded by sturdy bla
 The linear actuator that drives the shutter sits on the top of the structure and must be disconnected before the top is removed. 
 
 
-Setup
-=====
+Maintenance & Troubleshooting
```

Comment:
above you say "cardboard" it's actually called "black construction hardboard"

Reply:
done